trlx/trlx.py

```diff
@@ -90,4 +118,8 @@ def train(
         raise ValueError(f"Either {dataset=} or {reward_fn=} should be given")
 
     model.learn()
+
+    if accelerator.is_main_process and not ray.is_initialized():
+        accelerator.end_training()
```

Comment:
this is equivalent to `run.finish()` if you were to reuse that variable, but is this also a required thing since it was not present before.

Reply:
we can probably remove the `run` variable from above.